v4-client-js/examples/cancel_example.ts

```diff
@@ -0,0 +1,71 @@
+import { BECH32_PREFIX, OrderFlags } from '../src';
+import { CompositeClient } from '../src/clients/composite-client';
+import {
+  Network, OrderExecution, OrderSide, OrderTimeInForce, OrderType,
+} from '../src/clients/constants';
+import LocalWallet from '../src/clients/modules/local-wallet';
+import { Subaccount } from '../src/clients/subaccount';
+import { randomInt } from '../src/lib/utils';
+import { DYDX_TEST_MNEMONIC, MAX_CLIENT_ID } from './constants';
+
+async function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function test(): Promise<void> {
```

Comment:
Can you rename this cancel order example to `long_term_order_cancel_example`? Also we should make sure that no MMs onboarding to testnet reference this example. Maybe even add a comment in this example noting that programmatic traders should almost always be using short-term orders?

Maybe add this in a comment?

```
Programmatic traders should generally not use stateful orders for following reasons:

- Stateful orders received out of order by validators will fail sequence number validation and be dropped.
- Stateful orders have worse time priority since they are only matched after they are included on the block.
- Stateful order rate limits are more restrictive than Short-Term orders, specifically max 2 per block / 20 per 100 blocks.
- Stateful orders can only be canceled after they’ve been included in a block.
```